gravity-info-server/src/transactions.rs

```diff
@@ -144,6 +144,19 @@ impl From<&MsgTransfer> for CustomMsgTransfer {
     }
 }
 
+const DEVELOPMENT: bool = cfg!(feature = "development");
+const SSL: bool = !DEVELOPMENT;
+const DOMAIN: &str = if cfg!(test) || DEVELOPMENT {
+    "gravity-grpc.polkachu.com"
+} else {
+    "info.gravitychain.io"
+};
+const PORT: u16 = if cfg!(test) || DEVELOPMENT {
+    14290
+} else {
+    9000
+};
+
```

Comment:
Since i can only put declarations up here I decided on Polkachu reliance for development since this is the query and not where its being hosted.